src/migrations.zig

```diff
@@ -8,24 +8,41 @@ var arenaAllocator = main.utils.NeverFailingArenaAllocator.init(main.globalAlloc
 const migrationAllocator = arenaAllocator.allocator();
 
 var blockMigrations: std.StringHashMap([]const u8) = .init(migrationAllocator.allocator);
-
-pub fn registerBlockMigrations(migrations: *std.StringHashMap(ZonElement)) void {
-	std.log.info("Registering {} block migrations", .{migrations.count()});
-
+var biomeMigrations: std.StringHashMap([]const u8) = .init(migrationAllocator.allocator);
+
+const MigrationType = enum {
+	block,
+	biome,
+};
+
+pub fn registerAll(comptime typ: MigrationType, migrations: *std.StringHashMap(ZonElement)) void {
+	std.log.info("Registering {} biome migrations", .{migrations.count()});
```

Comment:
Should be `...{} {s} migrations", .{..., @tagName(typ)}`

Reply:
```suggestion
	std.log.info("Registering {} {s} migrations", .{migrations.count(), @tagName(typ)});
```